examples/import-boss/generators/import_restrict.go

```diff
@@ -192,8 +192,16 @@ func (importRuleFile) VerifyFile(f *generator.File, path string) error {
 	}
 
 	if rules == nil {
-		// No restrictions on this directory.
-		return nil
+		// No restrictions on this directory, double check under vendor.  Make an assumption about running in kube
+		vendorPath := strings.Replace(path, "/go/src/k8s.io/", "/go/src/k8s.io/kubernetes/vendor/k8s.io/", -1)
```

Comment:
Oh, the Import-path file here: https://github.com/kubernetes/kubernetes/pull/39384 isn't enforced because when I add it to the import-boss command, it looks in the wrong folder (not vendor) for the import-restriction file.  This forces it to check again in the correct directory, but isn't a reasonable solution.